api/appStore/deployment/wire_appStoreDeployment.go

```diff
@@ -27,7 +28,7 @@ import (
 	"github.com/google/wire"
 )
 
-var AppStoreDeploymentWireSet = wire.NewSet(
+var WireSet = wire.NewSet(
```

Comment:
have two wire sets here for EA and full mode and let's keep installedAppReader.* in these sets

Reply:
installedAppReader also has two wire sets. so they should be registered accordingly. AppStoreDeploymentWireSet is common for both EA and FULL mode. so registering `installedAppReader.EAWireSet` here